subt/docker/cudatest/cudatest3.py

```diff
@@ -0,0 +1,14 @@
+
+import torch
```

Comment:
What behavior do you want to get when torch (or tensorflow) is not installed at all?

Reply:
I envision a situation when torch or tensorflow is used inside a module, that is being loaded based on configuration file. The I am saying I want that module to run so crashing with import error is expected (by me at least :wink:). The same is true for example for opencv. So I wouldn't worry about it much.